Presentation.Web/app/components/global-admin/global-admin-organization-modal.view.html

```diff
@@ -23,7 +23,7 @@ <h3 class="modal-title">{{ ctrl.title }}</h3>
                 CVR nummer
             </label>
             <div class="col-xs-8">
-                <input type="text" ng-model="org.cvr" id="cvr" class="form-control" ng-required="false" ng-pattern="/^([a-zA-Z]{2})?([0-9]{8})$/">
+                <input type="text" ng-model="org.cvr" id="cvr" class="form-control" ng-required="false" data-element-type="orgModalCvrInput" ng-pattern="/^([a-zA-Z]{2})?([0-9]{8})$/">
```

Comment:
don't add data-element-type when there is a valid "id" property. Use the GetById in the tests in stead. Data-element-type was introduced to deal with kendo grid columns where we wanted to filter by column type. Unfortunately it was used elsewere as well, so that does not make it easy (for you) when extending it. But now you know when to use what :-)